query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/QueryRequestToPinotSQLConverter.java

```diff
@@ -271,7 +269,7 @@ private String convertExpressionToString(
             viewDefinition.getPhysicalColumnNames(getLogicalColumnName(expression));
         return joiner.join(columnNames);
       case ATTRIBUTE_EXPRESSION:
-        if (isMapSubpathAttributeExpression(expression, viewDefinition)) {
+        if (isMapAttributeExpression(expression, viewDefinition)) {
```

Comment:
regression - it was named correctly before - we need a subpath here, not any map.

Reply:
removed extra methods from the code now